staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -43,7 +43,7 @@ import (
 	utiltrace "k8s.io/utils/trace"
 )
 
-func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Interface, includeName bool) http.HandlerFunc {
+func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Interface) http.HandlerFunc {
```

Comment:
dropping this loses information about whether the particular path expects name information in the path. I think this should be reverted.